app/views/declaration/destinationCountries/change_routing_country.scala.html

```diff
@@ -50,37 +50,26 @@
 }
 }
 
-@autocomplete = {
-@components.fields.field_accessible_autocomplete(
-    form("countryCode"),
-    "",
-    None,
-    None,
-    messages("declaration.routingCountry.empty"),
-    AutoCompleteItem.fromCountry(Countries.allCountries, _.countryCode)
-)
-}
-
 @govukLayout(
     title = Title(s"declaration.routingCountry.title", "declaration.routingCountry.heading"),
     backButton = Some(BackButton(messages("site.back"), Navigator.backLink(ChangeCountryPage, mode)))){
 
     @formHelper(action = controllers.declaration.routes.RoutingCountriesSummaryController.submitChangeCountry(mode, countryToChange), 'autoComplete -> "off") {
         @errorSummary(form.errors)
 
-        @sectionHeader(messages("declaration.routingCountry.heading"))
+        @components.fields.field_accessible_autocomplete_with_header(
```

Comment:
This pattern was borrowed from destination_country (see below) - another page with a single question consisting of a country auto-complete list